languages/ruby/exercises/concept/basics/.docs/after.md

```diff
@@ -0,0 +1,42 @@
+Ruby is a dynamic and strongly typed language. In dynamic languages the type of a variable or object is resolved at runtime, which means that its value or type can be changed up to the very last moment (when it gets parsed by the interpreter).
+And what do we mean with strongly typed? Once we know the type of a variable or object, Ruby is strict about what you can do with it, for example:
+
+```ruby
+x = '2'
+y = x + 'n'
+=>  '2n'
+```
+
+**But**
+
+```ruby
+x = '2'
+y = x + 2
+=> TypeError (no implicit conversion of Integer into String)
+
+Remember, in Ruby everything is an object. Even classes are instances of the class `Class`. For example:
+
+```ruby
+1.class
+=> Integer
+
+Integer.instance_of?(Class)
+=> true
```

Comment:
```suggestion
Integer.class
=> Class
```

This is a more direct example of Integer class being an instance of `Class`.

But if we cant to communicate that everything is an object, then we can do:

```ruby
Integer.is_a?(Object)
=> true
```

Reply:
I like this:
```
Integer.is_a?(Object)
# => true

Class.is_a?(Object)
# => true
```

Reply:
I've added a commit doing what I suggested 👍